config/katello.migrations/160425017684-default-http.rb

```diff
@@ -0,0 +1,3 @@
+answers['foreman_proxy']['http'] = false if answers['foreman_proxy'].is_a? Hash
+
+answers['foreman_proxy'][('http' || 'foreman_proxy_http')] = answers['capsule'].delete('http') { |k| false } if answers['capsule'].is_a? Hash
```

Comment:
What is the `'http' || 'foreman_proxy_http'` supposed to be doing? That will always resolve to http as non-blank strings are truthy.